repair_tests/repair_test.py

```diff
@@ -1195,6 +1195,40 @@ def run_repair():
         else:
             node1.nodetool('repair keyspace1 standard1 -inc -par')
 
+    @since('3.0')
+    def repair_one_node_cluster_test(self):
```

Comment:
```
pytest  --cassandra-dir=$CASSANDRA_DIR repair_tests/repair_test.py --collect-only
```

pytest requires test methods to start with `test_`:
https://docs.pytest.org/en/reorganize-docs/new-docs/user/naming_conventions.html

Reply:
In result those added test cases are not run at all.
Did they run for You and passed @k-rus  ?

Reply:
I have no idea. I haven't found information how to build in Stargazer Jenkins. I asked in the channel, but it didn't get good enough reply. So I have to learn through the reviewer :)
Thank you for your feedback!

Reply:
I would start with manual run.

Reply:
Note: I think it would be good to start with build locally and start tests locally.